ultralytics/data/explorer/explorer.py

```diff
@@ -466,3 +466,71 @@ def generate_report(self, result):
         TODO
         """
         pass
+
+    def label_count(self, labels: list) -> dict:
+        """
+        Count of Occurrences of labels
+        Args:
+            labels (list): List of labels to count their occurrences.
+
+        Returns:
+            label_counts(dict): Dictionary containing labels with their count
+
+        Example:
+            ```python
+                exp = Explorer()
+                exp.create_embeddings_table()
+                label_counts = exp.label_count()
+            ```
+        """
+        label_counts = {}
+        dataset_labels = self.table.to_pandas().labels
+        flatten_labels = [l for ls in list(dataset_labels) for l in ls]
+        for label in labels:
+            if label in flatten_labels:
+                label_counts[label] = flatten_labels.count(label)
+            else:
+                label_counts[label] = flatten_labels.count(label)
+        return label_counts
+
+    def dataset_label_count(self) -> dict:
+        """
+        Count of Occurrences of all labels in dataset.
+
+        Returns:
+            label_counts(dict): Dictionary containing labels with their count
+
+        Example:
+            ```python
+                exp = Explorer()
+                exp.create_embeddings_table()
+                label_count = exp.dataset_label_count()
+            ```
+        """
+        label_counts = {}
+        dataset_labels = self.table.to_pandas().labels
```

Comment:
Access one column like this.. It'll not convert the entire table to pandas and hence will not load the entire thing into memory. So less chance of going OOM
```
table.to_lance().to_table(columns=["labels"]).to_pandas()
```
For other places as well

Reply:
ok, I'll make changes